conversion/requests/chatgpt/convert.go

```diff
@@ -22,12 +22,15 @@ func ConvertAPIRequest(api_request official_types.APIRequest) chatgpt_types.Chat
 			fmt.Println("Error getting Arkose token: ", err)
 		}
 		chatgpt_request.Model = api_request.Model
+		// Cover some models like gpt-4-32k
+		if len(api_request.Model) >= 7 && api_request.Model[6] >= 48 && api_request.Model[6] <= 57 {
+			chatgpt_request.Model = "gpt-4"
+		}
```

Comment:
just check if model is equal to gpt-4-32k. too convoluted

Reply:
See [doc](https://platform.openai.com/docs/models/model-endpoint-compatibility), the model could be gpt-4, gpt-4-0613, gpt-4-32k, gpt-4-32k-0613

Reply:
how about we just map everything starting with gpt-4 to gpt-4?

Reply:
There is some model like gpt-4-browsing, so that's not a good idea

Reply:
`api_request.Model[6] >= 48 && api_request.Model[6] <= 57`

What does this check for?

Reply:
the char after `gpt-4-` is a number